lib/generators/maintenance_tasks/templates/task_spec.rb.tt

```diff
@@ -5,8 +5,9 @@ module <%= tasks_module %>
 <% module_namespacing do -%>
   RSpec.describe <%= class_name %>Task do
     # describe '#process' do
+      # subject(:process) { described_class.process(element) }
       # it 'performs a task iteration' do
-        # <%= tasks_module %>::<%= class_name %>Task.process(element)
+        # expect something here
```

Comment:
Should we make this also more useful? This line is not going to be directly usable either.